h2o-r/h2o-package/R/models.R

```diff
@@ -565,6 +565,10 @@ h2o.performance <- function(model, newdata=NULL, train=FALSE, valid=FALSE, xval=
 
     ####
     # FIXME need to do the client-side filtering...  PUBDEV-874:   https://0xdata.atlassian.net/browse/PUBDEV-874
+    if (length(res$model_metrics) == 0) {
+      print("WARNING: Model metrics cannot be calculated and metric_json is empty due to the absence of the response column in your dataset.")
```

Comment:
But do we really know why are the metrics not defined? This statement is just guessing - it is the most probable cause, yes, but we do not know that for sure.

Reply:
Michal checked himself and verified that this is the case.  Thank you dear.